Configuration/Eras/python/Era_Run3_cff.py

```diff
@@ -13,6 +13,7 @@
 from Configuration.Eras.Modifier_run3_nanoAOD_devel_cff import run3_nanoAOD_devel
 from Configuration.Eras.Modifier_run3_egamma_cff import run3_egamma
 from Configuration.Eras.Modifier_run3_RPC_cff import run3_RPC
+from Configuration.Eras.Modifier_run3_HLTconditions_cff import run3_HLTconditions
 
-Run3 = cms.ModifierChain(Run2_2018.copyAndExclude([run2_GEM_2017, ctpps_2018]), run3_common, run3_egamma, run3_GEM, run3_HB, run3_HFSL, stage2L1Trigger_2021, ctpps_2022, dd4hep, run3_nanoAOD_devel, run3_RPC)
+Run3 = cms.ModifierChain(Run2_2018.copyAndExclude([run2_GEM_2017, ctpps_2018]), run3_common, run3_egamma, run3_GEM, run3_HB, run3_HFSL, stage2L1Trigger_2021, ctpps_2022, dd4hep, run3_nanoAOD_devel, run3_RPC, run3_HLTconditions)
```

Comment:
I believe the `run2_HLTconditions_2018` would need to be excluded here
```suggestion
Run3 = cms.ModifierChain(Run2_2018.copyAndExclude([run2_GEM_2017, ctpps_2018, run2_HLTconditions_2018]), run3_common, run3_egamma, run3_GEM, run3_HB, run3_HFSL, stage2L1Trigger_2021, ctpps_2022, dd4hep, run3_nanoAOD_devel, run3_RPC, run3_HLTconditions)
```

Reply:
Sure, this can be done. What about other modules using the ```run2_HLTconditions_2018``` ? 

I see it is used in [DQMOffline/Trigger/python/TopMonitoring_cff.py](https://github.com/cms-sw/cmssw/blob/master/DQMOffline/Trigger/python/TopMonitoring_cff.py) and [DQMOffline/Trigger/python/BTagAndProbeMonitoring_cff.py](https://github.com/cms-sw/cmssw/blob/master/DQMOffline/Trigger/python/BTagAndProbeMonitoring_cff.py) but excluding it should have no impact because the first one has the same default configuration and the second one only imports it but does not make use of it. 

Now I see that  ```run2_HLTconditions_2018``` is also used in [Configuration/StandardSequences/python/Eras.py](https://github.com/cms-sw/cmssw/blob/master/Configuration/StandardSequences/python/Eras.py#L84). I would add the ```run3_HLTconditions``` there, too. 

And should we do something here [PhysicsTools/NanoAOD/python/triggerObjects_cff.py](https://github.com/cms-sw/cmssw/blob/master/PhysicsTools/NanoAOD/python/triggerObjects_cff.py#L305) ?

Reply:
I'm no expert of Eras (nor polices related to them), but I know the HLT menu for pp collisions in recent CMSSW releases is one and is for Run 3 (ignoring developments related to Phase 2, which are irrelevant for this PR).

My vague recollection of these Run-2 modifiers (and maybe I'm wrong) is that they were related to Run-2 UL MCs, where the (offline-)DQM step would be done in a single release (10_6_X) for 2016+2017+2018, while the HLT step was done in different releases for the 3 different years; this meant that the HLT-DQM in 10_6_X needed to be able to adapt to different HLT Paths (from different menus).

Going back to the present (12_4_X and 12_5_X), the UL use case does not apply, we have the Run-3 HLT in the release, and my understanding is that its clients (DQM, nanoAOD, etc) should simply be configured accordingly.

In short, I don't see the need to have `run3_HLTconditions` (instead, I'd configure `zcounting` directly for the Run-3 HLT in the `cff`, which should be named `cfi` by the way), while indeed the ` run2_HLTconditions_2018` should be removed from the `Run3` era.

But if I'm missing something, please let me know.

Reply:
Many thanks for the reminder. I agree that the better option is simply removing the ```run2_HLTconditions_2018``` from the ```Run3```. I will do this in another PR.